### PR TITLE
Updated deployment target to 12.0 to match the SDK.

### PR DESCRIPTION
**Pull request by mhdostal**

### CODE REVIEW

**Comment by mhdostal** [comment]
@philium Thanks for catching that.  I made the change and also updated both versions to 100.8.

**Comment by philium** [comment]
Thanks for making those changes! Moving to a deployment target of iOS 12.0 causes two new warnings to be generated:

image

Were you planning on addressing those in a separate PR?

**Comment by mhdostal** [comment]
No, I will address them here.  Thanks for pointing those out, I missed them.

**Comment by mhdostal** [comment]
@philium Fixed the deprecations.

File: README.md [hunk 1]
```diff
@@ -16,10 +16,10 @@ Toolkit components that will simplify your iOS app development with ArcGIS Runti
 * [Bookmarks](Documentation/Bookmarks)
 
 ## Requirements
-* [ArcGIS Runtime SDK for iOS](https://developers.arcgis.com/en/ios/) 100.7.0 (or higher)
+* [ArcGIS Runtime SDK for iOS](https://developers.arcgis.com/en/ios/) 100.8.0 (or higher)
 * Xcode 10.2 (or higher)
```
Comment (esreli) [inline README.md:20]: Conveniently, we're having a similar conversation over in the Tutorials repo. I'm not the authoritative source here but @nixta has [informed us](https://github.com/ArcGIS/DevLabs/pull/43#discussion_r405002119) that as of 100.8 we are dropping support for Xcode 10. You may want to figure out the details here and adjust this line.

Comment (mhdostal) [reply README.md:20]: Ok, will do.  Thanks.

Comment (philium) [reply README.md]: I guess that means the toolkit can now take advantage of features [introduced](https://swift.org/blog/swift-5-1-released/) in Swift 5.1!

**Pull request merged**